src/pilot/deploy-overcloud.py

```diff
@@ -328,6 +1127,12 @@ def main():
             raise ValueError("\nError: {} failed, unable to continue.  See "
                              "the comments in that file for additional "
                              "information".format(cmd))
+        # Pass the parameters required by puppet which will be used
+        # to enable/disable dell nfv features
+        # Edit the dellnfv_environment.yaml
+        edit_dell_environment_file(args.enable_hugepage, args.enable_numa,
+                                   args.hugepage_size, vcpu_pin_set,
```

Comment:
i suspect this will fail if those params are not passed  as args, since they dont have a default value